accepted/future-releases/extension-types/feature-specification.md

```diff
@@ -1127,6 +1127,12 @@ A compile-time error occurs if any direct or indirect superinterface of
 _DV_ denotes the declaration _DV_. *As usual, subtype cycles are not
 allowed.*
 
+It is a compile-time error if two elements in the type list of
+the `implements` clause of an extension type specifies the same type.
```

Comment:
extension type -> extension type declaration

I really want to make it explicit that it's what is about what is written in the declaration, to avoid issues like we've had (has?) with mixin applications and desugaring.

Reply:
Of course! Done.